client/src/components/WatchList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class WatchList extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      beenClicked: false,
+      add: '',
+      remove: '',
+    };
+  }
+
+  render() {
+    return (
+      <button>Remove From Watchlist</button>
+    );
+  }
+}
+
+export default WatchList;
+
+// this.state is going to be conditionally rendered, changing the button from 'Add to Watchlist' to 'Remove From Watchlist' on click
+// it needs a function and a value for add and a value for remove.
```

Comment:
You should add a blank line at the end of each component for styling purposes.

Reply:
I'll get to it